src/types.ts

```diff
@@ -197,6 +197,8 @@ export interface ExcalidrawProps {
   UIOptions?: UIOptions;
   detectScroll?: boolean;
   handleKeyboardGlobally?: boolean;
+  resetLibrary?: () => void;
```

Comment:
is this needed? Can we reuse `addToLibrary` for this, passing an empty array? (I'd rename the `addToLibrary` to `saveLibrary`)

Reply:
good point 👍